packages/terra-tabs/src/common-tabs/Tab.module.scss

```
@@ -127,6 +129,15 @@
     }
   }
 
+  .button:focus {
+    box-shadow: var(--terra-tabs-add-button-focus-box-shadow, none);
```

```suggestion
    box-shadow: none;
```
I don't think this should be themable

---

Done https://github.com/cerner/terra-framework/commit/fcb1925802030dfb5be6bf821a0a978a5c82d40c